js/ad_manager_template.js

```diff
@@ -5,7 +5,9 @@
  * version 0.1
  */
 
-OO.Ads.manager(function(_, $) {
+const { bind } = require('underscore');
```

Comment:
Is there an option to use built-in bind func or just arrow functions to get rid of underscore's bind?

Reply:
Sure, but not in this PR

Reply:
Why not in this PR? Ticket called "Remove Underscore and Jquery dependencies from html5-ad-plugins". I guess if underscore is still required, it should be explained.